test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json

```diff
@@ -7601,13 +7541,6 @@
             "path": "system.io.pipelines/4.7.0",
             "hashPath": "system.io.pipelines.4.7.0.nupkg.sha512"
         },
-        "System.IO.Pipes/4.3.0": {
-            "type": "package",
-            "serviceable": true,
-            "sha512": "sha512-wpGJuACA6r8+KRckXoI6ghGTwgPRiICI6T7kgHI/m7S5eMqV/8jH37fzAUhTwIe9RwlH/j1sWwm2Q2zyXwZGHw==",
-            "path": "system.io.pipes/4.3.0",
-            "hashPath": "system.io.pipes.4.3.0.nupkg.sha512"
-        },
```

Comment:
I think this one is fine as well as it's not something we ship.

Reply:
This is a framework dependency now, correct?  We may need to verify version and if lower, have an explicit reference.

Reply:
Shared Fx has lower versions:

```
System.IO.Pipes.dll :  4.1.2.0
System.Threading.Overlapped.dll  : 4.1.2.0
```
Added explicit references to System.IO.Pipes  and System.Threading.Overlapped